### PR TITLE
Merge APIs

### PR DESCRIPTION
**Pull request by bholagabbar**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -3,14 +3,18 @@
...
+import bridgeRouter from "./routes/beta/bridge.js";
+import swapRouter from "./routes/beta/swap.js";
+import v1Router from "./routes/v1/index.js";
 
 const app = express();
 const port = process.env.port ?? 8080;
 
-app.use("/api/v1/bridge", bridgeRouter);
-app.use("/api/v1/swap", swapRouter);
+app.use("/beta/bridge", bridgeRouter);
```
Comment (caesar0x) [inline server/index.js:13]: curious, why did you change name of these routes from `/api/v1` => `/beta`

File: server/routes/v1/index.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        } catch (err) {
+            res.status(400).json({"error": "Valid arguments for fromChainId, toChainId, fromCoin, toCoin and inputTokenAmount must be passed"});
+            return;
+        }
+
+        const fromChainId = req.query.fromChainId;
+        const toChainId = req.query.toChainId
+        const fromToken = req.query.fromToken
+        const toToken = req.query.toToken
+        const amount = req.query.amount
```
Comment (caesar0x) [inline server/routes/v1/index.js]: btw, prob should make this:
`const {fromChainId, toChainId, toToken, amount} = req.query`
its cleaner esp for a bunch of params

File: server/index.js (hunk 1 above)
Comment (bholagabbar) [reply server/index.js:13]: Moved the old api to `beta` cause I didn't want it to conflict with my v1. I'll get rid of it once I replicate all the functionality

File: server/core/networks.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    BOBA: 288,
+    MOONBEAM: 1284,
+    MOONRIVER: 1285,
+    ARBITRUM: 42161,
+    AVALANCHE: 43114,
+    AURORA: 1313161554,
+    HARMONY: 1666600000,
+}
+
+const ETH = {
```
Comment (caesar0x) [inline server/core/networks.js:16]: btw, conna merge this but way to structure this is prob with 
```
export const CHAIN_INFO_MAP = {
  [ChainId.ETH]: {
    chainId: ChainId.ETH,
    chainSymbol: 'ETH',
    chainName: 'Ethereum',
  },
  [ChainId.OPTIMISM]: {
    chainId: ChainId.OPTIMISM,
    chainSymbol: 'OPTIMISM',
    chainName: 'Optimism',
  },
  [ChainId.BSC]: {
    chainId: ChainId.BSC,
    chainSymbol: 'BSC',
    chainName: 'Binance Smart Chain',
    chainShortName: 'BSC',
  },
  [ChainId.POLYGON]: {
    chainId: ChainId.POLYGON,
    chainSymbol: 'POLYGON',
    chainName: 'Polygon',
  },
  [ChainId.FANTOM]: {
    chainId: ChainId.FANTOM,
    chainSymbol: 'FANTOM',
    chainName: 'Fantom',
  },
  [ChainId.BOBA]: {
    chainId: ChainId.BOBA,
    chainSymbol: 'BOBA',
    chainName: 'Boba Network',
  },
  [ChainId.MOONBEAM]: {
    chainId: ChainId.MOONBEAM,
    chainSymbol: 'MOONBEAM',
    chainName: 'Moonbeam',
  },
  [ChainId.MOONRIVER]: {
    chainId: ChainId.MOONRIVER,
    chainSymbol: 'MOONRIVER',
    chainName: 'Moonriver',
  },
  [ChainId.ARBITRUM]: {
    chainId: ChainId.ARBITRUM,
    chainSymbol: 'ARBITRUM',
    chainName: 'Arbitrum',
  },
  [ChainId.AVALANCHE]: {
    chainId: ChainId.AVALANCHE,
    chainSymbol: 'AVALANCHE',
    chainName: 'Avalanche',
  },
  [ChainId.AURORA]: {
    chainId: ChainId.AURORA,
    chainSymbol: 'AURORA',
    chainName: 'Aurora',
  },
  [ChainId.HARMONY]: {
    chainId: ChainId.HARMONY,
    chainSymbol: 'HARMONY',
    chainName: 'Harmony',    
  },
}
```
